frontend/src/pages/articles/components/tags/tags.tsx

```diff
@@ -13,7 +16,19 @@ const Tags: React.FC<Properties> = ({ tags }) => (
           <span className={styles.categoryText}>{tag.category}</span>
         </div>
         <div className={styles.prompt}>
-          <p className={styles.promptText}>{tag.text}</p>
+          <BlockWithTooltip
+            tooltipContent={tag.text}
+            placement="bottom"
+            tooltipId={DataTooltipId.PROMPT_TAG_TOOLTIP}
+          >
+            <p className={styles.promptText}>{tag.text}</p>
+          </BlockWithTooltip>
+          <Tooltip
+            id={DataTooltipId.PROMPT_TAG_TOOLTIP}
+            shouldHideArrow
+            className={styles.promptTagTooltip}
+            offset={5}
```

Comment:
Check **B31**

Reply:
sorry, what does that mean?

Reply:
Of course, it's quality criteria:grimacing:
Thanks!